### PR TITLE
FutarchyChallengeFactory unit tests

### PR DESCRIPTION
**Pull request by ewilz**

### CODE REVIEW

File: test/unit/FutarchyChallengeFactory.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+contract('FutarchyChallengeFactory', (accounts) => {
+  let futarchyChallengeFactory
+
+  const mockAddresses = [
+    '0x4e0100882b427b3be1191c5a7c7e79171b8a24dd',
+    '0x6512df5964f1578a8164ce93a3238f2b11485d1c',
+    '0x687355ca7a320e5420a3db5ae59ef662e4146786'
+  ]
+
+  let COMPARATOR_TOKEN, SCALAR_PRICE_ORACLE_FACTORY, DUTCH_EXCHANGE // async "const's"
```
Comment (ewilz) [inline test/unit/FutarchyChallengeFactory.js:22]: subpar solution :/

Comment (cwhinfrey) [reply test/unit/FutarchyChallengeFactory.js:22]: Why not set these here instead of deployChallengeFactory if they're only deployed once anyway?

**Review by cwhinfrey — APPROVED** [review]
👍 Looks good!

File: test/unit/FutarchyChallengeFactory.js (hunk 1 above)
Comment (ewilz) [reply test/unit/FutarchyChallengeFactory.js:22]: Doing async transactions at the `describe` or `contract` level seems to make things wonky, including wonky test logging and you can't call `only` on any test clauses.

**Pull request merged**